lambdas/search/index.py

```diff
@@ -50,6 +73,26 @@ def lambda_handler(request):
         }
         size = 0
         _source = []
+    elif action == 'images':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": IMG_EXTS
+                }
+            }
+        }
+        size = 10
```

Comment:
both `size = ` values should be set by GLOBALS, e.g. `NUM_PREVIEW_IMAGES`, `NUM_PREVIEW_FILES`